wp-content/plugins/civicrm/civicrm/ext/afform/core/ang/af/afForm.component.js

```
@@ -382,6 +382,29 @@
         }
 
+        // PU Visit Date validation
+        if (ctrl.getFormMeta().name === 'afformMaterialContributionForPUCopy') {
+          var visitDateField = $element.find("af-field[name='activity_date_time'] input[type='text']");
+          if (visitDateField.length) {
+              var visitDateValue = visitDateField.val().trim();
+              if (visitDateValue !== "") {
+                  var visitDateParts = visitDateValue.split('/');
+                  if (visitDateParts.length === 3) {
+                      var visitDate = new Date(visitDateParts[2], visitDateParts[1] - 1, visitDateParts[0]);
+                      var today = new Date();
```

Biome

[error] Shouldn't redeclare 'today'. Consider to delete it or rename it.

'today' is defined here:


(lint/suspicious/noRedeclare)